sim/dex-data.ts

```diff
@@ -756,6 +758,7 @@ export class Species extends BasicEffect implements Readonly<BasicEffect & Speci
 		this.requiredItem = data.requiredItem || undefined;
 		this.requiredItems = this.requiredItems || (this.requiredItem ? [this.requiredItem] : undefined);
 		this.baseStats = data.baseStats || {hp: 0, atk: 0, def: 0, spa: 0, spd: 0, spe: 0};
+		this.bst = Object.values(this.baseStats).reduce((x, y) => x + y);
```

Comment:
This is an inner loop, so I'd rewrite this for performance.

```suggestion
		this.bst = this.baseStats.hp + this.baseStats.atk + this.baseStats.def + this.baseStats.spa + this.baseStats.spd + this.baseStats.spe;
```

Reply:
Should I do this for the other instances where I reassign?

Reply:
Reassign? I didn't see any other instances, but yes, sure.

Reply:
I might've used the wrong word. In some of the chat commands I assign a new BST to the cloned Species.